conf/messages.en

```diff
@@ -135,8 +135,8 @@ registerBusinessRepresenting.town = Town or city
 registerBusinessRepresenting.postCode = Postcode
 registerBusinessRepresenting.country = Country
 registerBusinessRepresenting.checkYourAnswersLabel =  Their registered details
-registerBusinessRepresenting.error.eoriNumber.required = Enter their EORI number
-registerBusinessRepresenting.error.eoriNumber.format = Enter their EORI number in the correct format
+registerBusinessRepresenting.error.eoriNumber.required = Enter the EORI number of the business, organisation or individual that you're representing 
```

Comment:
wrong character ' . replace it for you’re

Reply:
Needs to be a smart apostrophe e.g. you’ll .  (Alt=Shift-} on Mac keyboard)